client/layout/guided-tours/tours/jetpack-monitoring-tour.js

```diff
@@ -0,0 +1,83 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+export const JetpackMonitoringTour = makeTour(
+	<Tour name="jetpackMonitoring" version="20180611" path="/non-existent-route" when={ noop }>
```

Comment:
Is `when` necessary? Doesn't appear to be required via `Tour.propTypes`.

Can you explain this `path="/non-existent-route"`?

Reply:
> Is `when` necessary? Doesn't appear to be required via `Tour.propTypes`.

Copy-past. Dropped it, still seems to work.

> Can you explain this `path="/non-existent-route"`?

Copy-pasta :flushed: Also scrapped.